docs/source/config/encoders/esjson.rst

```diff
@@ -36,7 +36,7 @@ Config:
     indexed into ElasticSearch. Available fields to choose are "Uuid",
     "Timestamp", "Type", "Logger", "Severity", "Payload", "EnvVersion", "Pid",
     "Hostname", and "Fields" (where "Fields" causes the inclusion of any and
```

Comment:
I think it's worth it because people's config's won't break when the load this config. Also, how will people choose to send all Fields (the current default) without this behavior?